enterprise-suite/scripts/install-es.sh

```diff
@@ -196,7 +196,7 @@ if [ "false" != "$ES_EXPORT_YAML" ]; then
 
     if [ -z "$ES_LOCAL_CHART" ]; then
         TDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'install-es-tdir')
-        debug helm fetch -d $TDIR "$VERSION_ARG" "$chart_ref"
+        debug helm fetch -d $TDIR --devel "$VERSION_ARG" "$chart_ref"
```

Comment:
according to man page --devel option is ignored when version is specified.

Reply:
Yeah.  `$VERSION_ARG` will have a value if and only if the user specifies a value on the `install-es.sh` command line.  If they do, `--devel` will be ignored.  If they don't, it'll be empty and `--devel` will come into play.